starter-code/index.js

```diff
@@ -1,11 +1,58 @@
 class SortedList {
-    constructor() {}
-    add(item) {}
-    get(pos) {}
-    max() {}
-    min() {}
-    average() {}
-    sum() {}
-  };
-  
-  module.exports = SortedList;
\ No newline at end of file
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
+  add(x) {
+    this.items.push(x);
+    this.length = this.items.length;
+    this.items.sort((el1, el2) => {
+      if (el1 > el2) return 1;
```

Comment:
I know I told you to write it like that but here is the way to write less ;)
this.items = this.items.sort((a, b) => a-b)